apps/jclock/app.js

```diff
@@ -0,0 +1,68 @@
+    46,
+    atob("GBo2NjY2NjY2NjY2Gg=="),
+    94+(scale<<8)+(1<<16)
+  );
+  return this;
+};
+
+var drawTimeout;
+
+// schedule a draw for the next minute
+function queueDraw() {
+  if (drawTimeout) clearTimeout(drawTimeout);
+  drawTimeout = setTimeout(function() {
+    drawTimeout = undefined;
+    draw();
+  }, 60000 - (Date.now() % 60000));
+}
+
+const zeroPad = (num, places) => String(num).padStart(places, '0');
+
+function draw() {
+  let barWidth = 64;
+  let date = new Date();
+  
+  // queue next draw in one minute
+  queueDraw();
+  
+  // clean screen
+  g.reset().clearRect(Bangle.appRect);
+  
+  // draw side bar in blue
+  g.setColor('#00f');
+  g.fillRect(0, 0, barWidth, g.getHeight());
+  
+  // show time on the right
+  g.setColor('#000');
```

Comment:
Hardcoded black color makes the time invisible on dark theme. Please use e.g. the graphics theme object https://www.espruino.com/ReferenceBANGLEJS2#l_Graphics_theme , e.g. ` g.setColor(g.theme.fg);` should work I think.

Reply:
Oh, I don't use the dark theme so I didn't notice that. I will update it. Thanks.